frontend/packages/console-plugin-sdk/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "@console/plugin-sdk",
+  "version": "0.0.0-fixed",
+  "description": "Console static plugin SDK",
+  "private": true,
+  "main": "src/index.ts",
+  "scripts": {
+    "test": "yarn --cwd ../.. run test packages/console-plugin-sdk"
```

Comment:
What about e2e (integration tests), where do they fit?
Will we still collocate them on a single place considering they test whole application? Or split them per plugin?

The first option sounds better to me.

Reply:
> What about e2e (integration tests), where do they fit?
> Will we still collocate them on a single place considering they test whole application?

Yes, I think that's the general consensus. Unit tests, on the other hand, should be co-located with their corresponding package.